UserManagement/src/main/java/com/joe/trading/user_management/service/impl/UserServiceImpl.java

```diff
@@ -21,4 +23,10 @@ public User createUser(User user) {
     public User getUserById(Long userId) {
         return userRepository.getReferenceById(userId);
     }
+
+    @Override
+    public List<User> getAllUsers() {
+        List<User> users = userRepository.findAll();
+        return users.stream().toList();
```

Comment:
Since `userRepository.findAll()` returns a list, calling users.stream.toList() is redundant. 
Is there a specific reason you used this?